src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form "
+    data-qa="big"
+    action="#"
+    method="post">
```

Comment:
1) apply code styling recommendations. For similar cases as well
2) remove extra space class="form " -> class="form"
```suggestion
    <form
      class="form"
      data-qa="big"
      action="#"
      method="post"
    >
```